diff -Naur orpie-1.5.2.orig/gsl/gsl_sf.ml orpie-1.5.2/gsl/gsl_sf.ml
--- orpie-1.5.2.orig/gsl/gsl_sf.ml	2017-07-09 19:33:38.174131826 -0300
+++ orpie-1.5.2/gsl/gsl_sf.ml	2017-07-09 19:34:15.951693502 -0300
@@ -1142,34 +1142,6 @@
 
 
 
-(* Associated LEGENDRE functions *)
-external legendre_Plm : int -> int -> float -> float
-    = "ml_gsl_sf_legendre_Plm"
-
-external legendre_Plm_e : int -> int -> float -> result
-    = "ml_gsl_sf_legendre_Plm_e"
-
-
-external  legendre_Plm_array : int -> int -> float -> float array -> unit 
-    = "ml_gsl_sf_legendre_Plm_array"
-
-
-external legendre_sphPlm : int -> int -> float -> float
-    = "ml_gsl_sf_legendre_sphPlm"
-
-external legendre_sphPlm_e : int -> int -> float -> result
-    = "ml_gsl_sf_legendre_sphPlm_e"
-
-
-external  legendre_sphPlm_array : int -> int -> float -> float array -> unit 
-    = "ml_gsl_sf_legendre_sphPlm_array"
-
-
-external  legendre_array_size : int -> int -> int 
-    = "ml_gsl_sf_legendre_array_size"
-
-
-
 (* LOGARITHM and related functions *)
 external log : float -> float
     = "ml_gsl_sf_log" "gsl_sf_log" "float"
diff -Naur orpie-1.5.2.orig/gsl/mlgsl_sf.c orpie-1.5.2/gsl/mlgsl_sf.c
--- orpie-1.5.2.orig/gsl/mlgsl_sf.c	2017-07-09 19:33:38.174131826 -0300
+++ orpie-1.5.2/gsl/mlgsl_sf.c	2017-07-09 19:34:33.532109943 -0300
@@ -451,43 +451,6 @@
 SF1(legendre_Q1, Double_val)
 SF2(legendre_Ql, Int_val, Double_val)
 
-/* Associated Legendre Polynomials and Spherical Harmonics  */
-SF3(legendre_Plm, Int_val, Int_val, Double_val) 
-CAMLprim value 
-ml_gsl_sf_legendre_Plm_array(value lmax, value m, value x, value result_array)
-{
-  gsl_sf_legendre_Plm_array(Int_val(lmax),
-                            Int_val(m),
-                            Double_val(x),
-                            Double_array_val(result_array));
-  return Val_unit;
-}
-
-SF3(legendre_sphPlm, Int_val, Int_val, Double_val) 
-CAMLprim value 
-ml_gsl_sf_legendre_sphPlm_array(value lmax, value m, 
-                                value x, value result_array)
-{
-  gsl_sf_legendre_sphPlm_array(Int_val(lmax),
-                               Int_val(m),
-                               Double_val(x),
-                               Double_array_val(result_array));
-  return Val_unit;
-}
-
-CAMLprim value
-ml_gsl_sf_legendre_array_size(value lmax, value m)
-{
-  CAMLparam2(lmax, m);
-  CAMLlocal1(ret);
-  int gsl_ret;
-
-  gsl_ret = gsl_sf_legendre_array_size(Int_val(lmax), Int_val(m));
-  
-  ret = Val_int(gsl_ret);
-  CAMLreturn(ret);
-}
-
 /* LOGARITHM and related functions */
 SF1(log, Double_val)
 SF1(log_abs, Double_val)
