diff -Naur orpie-1.5.2.orig/gsl/mlgsl_error.c orpie-1.5.2/gsl/mlgsl_error.c
--- orpie-1.5.2.orig/gsl/mlgsl_error.c	2017-07-09 19:33:38.170798414 -0300
+++ orpie-1.5.2/gsl/mlgsl_error.c	2017-07-09 19:38:27.188607250 -0300
@@ -34,6 +34,7 @@
 
 static void ml_gsl_raise_exn(const char *msg, int gsl_errno)
 {
+  CAMLparam0();
   CAMLlocal2(exn_msg, exn_arg);
   exn_msg = copy_string(msg);
   exn_arg = alloc_small(2, 0);
