diff -Naur orpie-1.5.2.orig/gsl/mlgsl_sf.c orpie-1.5.2/gsl/mlgsl_sf.c
--- orpie-1.5.2.orig/gsl/mlgsl_sf.c	2017-07-09 19:33:38.174131826 -0300
+++ orpie-1.5.2/gsl/mlgsl_sf.c	2017-07-09 19:37:44.361377346 -0300
@@ -258,7 +258,7 @@
 SF3(ellint_F, Double_val, Double_val, GSL_MODE_val)
 SF3(ellint_E, Double_val, Double_val, GSL_MODE_val)
 SF4(ellint_P, Double_val, Double_val, Double_val, GSL_MODE_val)
-SF4(ellint_D, Double_val, Double_val, Double_val, GSL_MODE_val)
+SF3(ellint_D, Double_val, Double_val, GSL_MODE_val)
 SF3(ellint_RC, Double_val, Double_val, GSL_MODE_val)
 SF4(ellint_RD, Double_val, Double_val, Double_val, GSL_MODE_val)
 SF4(ellint_RF, Double_val, Double_val, Double_val, GSL_MODE_val)
